index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
I'm not sure why you have a html file in this project... :thinking:

Reply:
I was coding on VScode while I link the Html with JavaScript file to open the console in the browser